docs/usage.rst

```diff
@@ -393,7 +393,10 @@ without a declared hash using ``accept_untrusted_content=True``:
 
 This will skip checking the hash of ``content`` and ``content_type`` only if
 the ``Authorization`` header omits the ``hash`` attribute. If the ``hash``
-attribute is present, it will be checked as normal.
+attribute is present, it will be checked as normal. For requests that have no
```

Comment:
Let's document this explicitly. Say that `content` can be an empty string or `content` can be `None`.